Makefile

```diff
@@ -70,7 +70,11 @@ snyk: FORCE
 
 docker-image: FORCE
 	docker build -t cndp --build-arg http_proxy=${http_proxy} \
-  --build-arg https_proxy=${http_proxy} -f containerization/docker/Dockerfile .
+  --build-arg https_proxy=${http_proxy} -f containerization/docker/ubuntu/Dockerfile .
+
+docker-fed-image: FORCE
```

Comment:
I could look into creating a bash script similar to cne_build.sh to streamline the docker builds in the future to avoid and explosion of docker-**_os_**-image and docker-**_os_**-run rules in the makefiles... what do we think?